package.json

```diff
@@ -53,18 +53,16 @@
     "sprintf-js": "^1.1.2"
   },
   "devDependencies": {
-    "@babel/cli": "^7.6.2",
```

Comment:
Looks like something weird happened here - all the dependencies got downgraded. 😬 You can either fix this up, or I can take a look at it once the other documentation changes are all ready 🙇

Reply:
Sorry Arthur, we're trying to do some rebasing to fix old commit messages and things aren't going as smoothly yet

We're still working on it :sweat_smile: